python/webhooks/README.md

```diff
@@ -1,3 +1,5 @@
+> Note: This legacy app is no longer supported. Please access the **webhooks_v5** directory for the most up-to-date webhooks app.
```

Comment:
Nit: We should probably link this `webhooks_v5` you can anticipate what the GitHub URL would be and add it now or add link later